src/System.Private.ServiceModel/src/System/ServiceModel/EnvelopeVersion.cs

```diff
@@ -18,6 +18,10 @@ public sealed class EnvelopeVersion
         private string[] _mustUnderstandActorValues;
         private string _senderFaultName;
         private string _receiverFaultName;
+        private const string Soap11ToStringFormat = "Soap11 ({0})";
```

Comment:
Nit: possibly add comment to explain why these are deliberately not in the resources.  Don't block on this suggestion.